deploy/common/pmem-storageclass-ext4-kata.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: pmem-csi-sc-ext4-kata
+parameters:
+  csi.storage.k8s.io/fstype: ext4
+  eraseafter: "true"
+  kataContainers: "true"
+provisioner: pmem-csi.intel.com
+reclaimPolicy: Delete
+volumeBindingMode: Immediate
```

Comment:
As per [app yaml(https://github.com/intel/pmem-csi/pull/500/files#diff-17203e3a5882efb0c1944558564a9c52R10-R11] , the application is expected to run only on nodes with `katacontainers.io/kata-runtime: "true"`. So, if we use immediate binding mode might end up creating a node on the wrong node?

Reply:
True. I should better switch this to "late-binding".